### ISSUE TITLE
Copy all the storage query result

### ISSUE THREAD

**Issue by xlc** [polkadot-api/papi-console#13]
I often need to query all the entries under a storage map and need to search for a particular key. It seems like the current page is implemented very efficiently that only visible items are rendered in DOM. But it is too efficient that I cannot use it. I need a way to dump all the JSON so I can search it

**Comment by josepot** [comment]
I see... Yeah, that's annoying... I guess that we could add a fuzzy-filter? 🤔 Also, it would probably be very useful to have the ability to just copy into the clipboard a JSON with all the results...

What do you think?

**Comment by xlc** [comment]
a button to just copy everything to clipboard will be great

**Comment by xlc** [comment]
also it seems that the keys are not displayed?

**Comment by xlc** [comment]
<img width="1106" alt="Screenshot 2024-11-28 at 10 18 14 PM" src="https://github.com/user-attachments/assets/19635941-27de-4131-9bf4-d28db600f596">
 not in this case

**Comment by carlosala** [comment]
Yep, we are not showing "complex" keys. It is indeed something missing.

**Comment by ggwpez** [comment]
Also hit this again, i want to copy all of the results as JSON...  It only shows to copy single values.  
Alternative is only to use PJS and then try to convert that to compliant json output.

![Image](https://github.com/user-attachments/assets/430cd6ab-e816-4956-8fd2-a1eff8b47d56)

**Comment by voliva** [comment]
I've just changed the JSON view so it shows the whole value as a JSON, keys included. The JSON display has a button to copy items at any level (also the whole value)

We might have to improve the decoded version, but it's not urgent anymore.
